packages/ui/src/components/ActionableBox/ActionableBox.tsx

```diff
@@ -49,7 +49,7 @@ export const ActionableBox = memo<ActionableBoxProps>(({
 			{...divProps}
 			className={componentClassName(null, className)}
 		>
-			<div className={componentClassName('contents')}>{children}</div>
```

Comment:
it is imho good, so you can target it better (e.g. in custom css)

Reply:
I understand, however:
- it adds complexity
- it is more ambiguous whether/how it should inherit container styles
- it is impossible to remove the extra container, on the other hand,
- it is easy to provide own container around children

Reply:
Reverted. Added `display: contents` instead.